### ISSUE TITLE
webpack для БЭМ?

### ISSUE THREAD

**Issue by nanervax** [bem-site/bem-forum-content-ru#1359]
Есть ли примеры использования webpack для БЭМ?

**Comment by tadatuta** [comment]
@nanervax смотря что именно подразумевается под «для БЭМ».
Например, [bem-react-core](https://github.com/bem/bem-react-core) собирается на Webpack.

**Comment by nanervax** [comment]
Имеется ввиду обычный БЭМ проект с использованием i-bem, хочется интегрировать с Webpack

**Comment by tadatuta** [comment]
Есть вот такой эксперимент https://github.com/bempack и [небольшой рассказ](https://ru.bem.info/forum/774/) про него, но оно дико медленное, так что as is использовать я бы не рекомендовал, но в качестве источника вдохновения для собственной разработки можно использовать.

**Comment by nanervax** [comment]
Верно ли я понимаю, что это плоды этой работы:
https://ru.bem.info/forum/774/

**Comment by tadatuta** [comment]
Да, я же как раз ссылку на эту страницу привел в своем предыдущем сообщении ;)
